tldw_Server_API/app/api/v1/endpoints/media/process_documents.py

```
@@ -0,0 +1,54 @@
+from tldw_Server_API.app.api.v1.endpoints import _legacy_media as legacy_media  # type: ignore
+
+router = APIRouter()
+
+
+@router.post(
+    "/process-documents",
+    summary="Extract, chunk, analyse Documents (NO DB Persistence)",
+    tags=["Media Processing (No DB)"],
+)
+async def process_documents_endpoint(
+    form_data: legacy_media.ProcessDocumentsForm = Depends(legacy_media.get_process_documents_form),
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)